src/InfoViz/Native/MutualInformationDiagram/index.js

```diff
@@ -187,7 +187,7 @@ function informationDiagram(publicAPI, model) {
 
     if (variableList.length < 2 || !model.container) {
       // Select the main circle and hide it and unhide placeholder
-      d3.select(model.container).select('svg.information-diagram').classed(style.hidden, true);
+      d3.select(model.container).select('svg.information-diagram').attr('class', style.informationDiagramSvgHide);
```

Comment:
Nope that's fine, and it was the only reason I could think of for using attr instead.  But I just wanted to be sure you did it on purpose.  Thanks :-)